fintech-examples/fintech-ui/src/app/login/login.component.html

```diff
@@ -1,13 +1,13 @@
 <form (ngSubmit)="onSubmit()" [formGroup]="loginForm">
-  <div class="card">
+  <div class="card card-frame">
```

Comment:
Is it a bootstrap class?

Reply:
Nope! A custom class I created for login and bank-search card. Take a look at style.scss when you want to know what this does